GirafRest/appsettings.Production.json

```diff
@@ -0,0 +1,40 @@
+﻿{
+    "ConnectionStrings": {
+        "DefaultConnection": "server=172.19.10.30;port=3306;userid=root;password=unknown;database=giraf-prod;Allow User Variables=True"
```

Comment:
Har hørt om grupper på forrige semestre som er blevet angrebet af github bots, som leder efter github repositories som gemmer deres database ConnectionStrings online, og hvis de finder den, automatisk sletter alt i hele databasen, og kræver penge for at give data retur. Ved ikke om det bliver et problem ved os, da jeg mindes at vores github repository kun kan ses af dem der er med i organisationen, men er ikke sikker. Uanset hvad, plejer det ikke at være en god ide at gemme den online